lib/AbiBinProvider.js

```diff
@@ -5,8 +5,8 @@ const fs = require('fs'),
   path = require('path');
 //__NOT_FOR_WEB__END__
 
-const Mosaic = require('@openstfoundation/mosaic-tbd');
-const AbiBinProvider = Mosaic.AbiBinProvider;
+const Mosaic = require('@openstfoundation/mosaic.js');
+const { AbiBinProvider } = Mosaic;
```

Comment:
Do you need `Mosaic` further down? If not you could merge the above two lines into:
```suggestion
const { AbiBinProvider } = require('@openstfoundation/mosaic.js');
```

Reply:
Done!